src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -169,3 +169,49 @@ bool Terminal::SetColorTableEntry(const size_t tableIndex, const DWORD dwColor)
     _buffer->GetRenderTarget().TriggerRedrawAll();
     return true;
 }
+
+// Method Description:
+// - Sets the cursor style to the given style.
+// Arguments:
+// - cursorStyle: the style to be set for the cursor
+// Return Value:
+// - true iff we successfully set the cursor style
+bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
+{
+    CursorType finalCursorType = CursorType::Legacy;
+    bool shouldBlink = false;
+
+    switch (cursorStyle)
+    {
+    case DispatchTypes::CursorStyle::BlinkingBlock:

```

Comment:
Please apply the [[[fallthrough]]](https://en.cppreference.com/w/cpp/language/attributes/fallthrough) attribute (new in C++17) as appropriate per the ISO C++ Guidelines (the guideline summary text is misleading, it also talks about empty cases):
[ES.78: Always end a non-empty case with a break](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es78-always-end-a-non-empty-case-with-a-break)

Reply:
Done.